modules/t/methodLinkSpeciesSet.t

```diff
@@ -110,6 +110,10 @@ subtest "Test Bio::EnsEMBL::Compara::DBSQL::MethodLinkSpeciesSet::new(ALL)", sub
     
     is ($classification, $mlss_classification);
 
+    my @genome_dbs = ($gdb1, $gdb2);
+    is ($method_link_species_set->find_pairwise_reference, @genome_dbs,
+        'Correct genome_dbs and order returned by find_pairwise_reference()');
+
```

Comment:
Might make more sense to create a whole new `subtest` for this, since the subtest you've added to really says that it's testing the `::new` method